treeherder/webapp/api/runnable_jobs.py

```diff
@@ -77,30 +77,29 @@ def list(self, request, project):
         for label, node in tc_graph.iteritems():
             build_platform = node['task']['extra']['treeherder']['build']['platform']
             job_type_name = node['task']['metadata']['name']
+            task_metadata = node['task']['metadata']
+            treeherder_options = node['task']['extra']['treeherder']
```

Comment:
Though perhaps for `job_type_name` just refer to it directly below, since the variable is only ever used once. eg:
```python
ret.append({
    # ...
    'job_type_name': task_metadata['name'],
    # ...
})
```